src/main/kotlin/tools/jackson/module/kotlin/KotlinModule.kt

```diff
@@ -1,13 +1,14 @@
-package com.fasterxml.jackson.module.kotlin
+package tools.jackson.module.kotlin
 
+import tools.jackson.core.json.PackageVersion
```

Comment:
Wrong! This is not the version to use here, MUST be Kotlin module's own version def.
(will fix post-merge)